src/aks-preview/azext_aks_preview/_help.py

```diff
@@ -359,6 +359,9 @@
         - name: --auto-upgrade-channel
           type: string
           short-summary: Specify the upgrade channel for autoupgrade. It could be rapid, stable, patch, node-image or none, none means disable autoupgrade.
+        - name: --node-os-upgrade-channel
+          type: string
+          short-summary: <TODO let Paul fill in>
```

Comment:
Don't forget to update this field.

Reply:
Leaving this as well as your other comment open, until Paul fills this in

Reply:
Same as https://github.com/Azure/azure-rest-api-specs/blob/5a4f7662cba770d045d500894a5565c176453cb3/specification/containerservice/resource-manager/Microsoft.ContainerService/preview/2022-11-02-preview/managedClusters.json#L5532

"Manner in which the OS on your nodes is updated. It could be none, unmanged, nodeimage or securitypatch"

Reply:
Added default in as well, corrected casing, and ordered alphabetically:
"Manner in which the OS on your nodes is updated. It could be NodeImage, None, SecurityPatch or Unmanaged (default Unmanaged)"